lib/pbench/cli/server/user_management.py

```diff
@@ -0,0 +1,205 @@
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import config_setup, pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server.database.database import Database
+from pbench.server.database.models.users import User
+
+
+# User create CLI
+@click.group("user_group")
+@click.version_option()
+@pass_cli_context
+def cli(ctx):
```

Comment:
`cli` is a _very_ generic name given the scope that this has.  Should that be `user_command_cli` or `pbench_cli` or something similar which describes its scope?